python_modules/event-pipeline-demo/event_pipeline_demo/configs_spark.py

```diff
@@ -0,0 +1,1686 @@
+'''NOTE: THIS FILE IS AUTO-GENERATED. DO NOT EDIT
+
```

Comment:
I also this it is good standard practice for generated file to include inline how they were produced (e.g. what script and what parameters)

Reply:
Cool, added